### PR TITLE
Created option to add new category (budgets)

### PR DESCRIPTION
**Pull request by ewelina-slepko**

### CODE REVIEW

File: src/app/home/shared/forms/budgets-form/budgets-form.component.html [hunk 1]
```diff
@@ -39,27 +39,27 @@
...
+                    class="custom-category"></text-input>
       </div>
     </div>
 
     <input type="checkbox" id="cycle-checkbox" name="cycle-checkbox" value="true" [checked]="repeatCycle"
            (click)="isRepeatCycleFieldSelect()">
     <label for="cycle-checkbox"><span class="span-text">Repeat cycle</span></label>
 
-    <div class="btn-container" [class]="insideBudgetCard ? 'additional-margin' : null" >
+    <div class="btn-container" [class]="insideBudgetCard ? 'additional-margin' : null">
```
Comment (FilipTalaga) [inline src/app/home/shared/forms/budgets-form/budgets-form.component.html:62]: `[class.additional-margin]="insideBudgetCard"`  or `[class="insideBudgetCard && 'additional-margin'"` should be enough here. No need for ternary.

File: src/app/home/shared/forms/budgets-form/budgets-form.component.scss [hunk 2]
```diff
@@ -130,6 +125,11 @@
   cursor: pointer;
 }
 
+.custom-category {
+  max-width: 4rem;
+  margin-top: 4px
```
Comment (FilipTalaga) [inline src/app/home/shared/forms/budgets-form/budgets-form.component.scss:130]: Missing semicolorz. Also wouldn't hurt to apply 0.25rem here.

File: src/app/home/shared/forms/budgets-form/budgets-form.component.ts [hunk 3]
```diff
@@ -64,8 +66,16 @@ export class BudgetsFormComponent implements OnInit {
       this.budget.repeatCycle = this.repeatCycle;
       this.budget.uid = this.authService.currentUser.uid;
       this.apiService.addBudget(this.budget);
-
-      this.isNewBudgetCardVisible = false;
+      
+      this.closeNewBudgetCardEmitter.emit();
```
Comment (FilipTalaga) [inline src/app/home/shared/forms/budgets-form/budgets-form.component.ts:70]: I can see a function below that does this. Consider using that function here to maintain single source of truth.

File: src/app/home/shared/forms/budgets-form/dtos.ts [hunk 4]
```diff
@@ -44,6 +44,11 @@ export const categories = [
     name: 'Charity',
     icon: 'fas fa-hand-holding-heart',
     isSelected: false
+  },
+  {
+    name: '',
```
Comment (FilipTalaga) [inline src/app/home/shared/forms/budgets-form/dtos.ts:49]: Why the sad nameless icon?

**Comment by ewelina-slepko** [comment]
Done!

**Pull request merged**